bench/Alice/Alice.csproj

```diff
@@ -1,18 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <IsPackable>false</IsPackable>
-    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net47</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net472</TargetFrameworks>
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
What's the advantage of setting the `PlatformTarget` here?

Reply:
This is taken from BenchmarkDotnet wiki, The benchmark will actually re-compile the code with different settings and execute them respectively

Reply:
Thanks for the info :+1: